ext/bcmath/bcmath.c

```diff
@@ -57,9 +57,32 @@ ZEND_TSRMLS_CACHE_DEFINE()
 ZEND_GET_MODULE(bcmath)
 #endif
 
+ZEND_INI_MH(OnUpdateScale)
+{
+	int *p;
+	zend_long tmp;
+#ifndef ZTS
+	char *base = (char *) mh_arg2;
+#else
+	char *base;
+
+	base = (char *) ts_resource(*((int *) mh_arg2));
+#endif
+
+	tmp = zend_atol(ZSTR_VAL(new_value), ZSTR_LEN(new_value));
+	if (tmp < 0 || tmp > INT_MAX) {
+		return FAILURE;
+	}
+
+	p = (int *) (base+(size_t) mh_arg1);
```

Comment:
```suggestion
	p = (int *) ZEND_INI_GET_ADDR();
```
after rebasing on https://github.com/php/php-src/commit/41c7d28c113d1c8e9c98cc834e8479f8567e5e3a.

Reply:
done